src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,43 @@
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from '../components/SearchLink';
+import classNames from 'classnames';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sex = searchParams.get('sex') || null;
+  const centuries = searchParams.getAll('centuries') || [];
+  const query = searchParams.get('query') || '';
+
+  const resetFilters = {
+    sex: null,
+    centuries: null,
+    query: null,
+  };
```

Comment:
The `resetFilters` object sets `sex`, `centuries`, and `query` to `null`. However, when using `setSearchParams`, setting a parameter to `null` will not remove it from the URL. Consider using an empty string or omitting the parameter entirely to effectively reset the filters.